.github/workflows/actuated-aarch64-test.yaml

```diff
@@ -0,0 +1,60 @@
+name: Actuated aarch64 test
+
+on: [push, pull_request]
+
+# Cancel any preceding run on the pull request.
+concurrency:
+  group: stream-test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/criu-dev' }}
+
+jobs:
+  build:
+    runs-on: actuated-arm64-4cpu-3gb
```

Comment:
The actuated CI tests only work for the main GitHub repository. When triggered in a forked repository, the job gets stuck with the following message:
```
Requested labels: actuated-arm64-4cpu-3gb
Job defined at: adrianreber/criu/.github/workflows/actuated-aarch64-test.yaml@refs/heads/2024-07-04-actuated
Waiting for a runner to pick up this job...
```
Could you please add something like the following to disable this workflow in forks?
```yaml
jobs:
  build:
    if: ${{ github.repository == 'checkpoint-restore/criu' }}
    runs-on: actuated-arm64-4cpu-3gb
```